apps/Conduit-UI/src/components/storage/StorageConfig.tsx

```diff
@@ -95,7 +96,7 @@ const StorageConfig: React.FC = () => {
                     {watchProvider === 'azure' && (
                       <Grid item xs={6}>
                         <FormInputText
-                          name="azure.connectionString"
+                          {...register('azure.connectionString')}
```

Comment:
We usually just use the name of a field as an indicator for React Hook Form so this adds some inconsistency, though it's a totally valid way of handling the forms

Reply:
@KostasFeg  When the name is misspelled, it will not be reminded of the error，For example, `google.serviceAccountKeyPath`. 

If it is for consistency, I can modify it back.